Makefile is not properly handling shellscript description

`Makefile` in the project root is not handling shellscript and is causing unexpected behaviour, such as:

* the check of existing cluster always fails because the value of `EXISTING_CLUSTER` is not passed to the `if` validation
* `RATING_SERVICE_ADDR` is not passed to the following line because of the lack of continuation

The error message around `RATING_SERVICE_ADDR`:

```
...
- deployment/frontend: container server terminated with exit code 2
    - pod/frontend-6595944855-rsw4m: container server terminated with exit code 2
      > [frontend-6595944855-rsw4m server] {"time":"2021-06-03T23:51:53.773692059Z","updates":null}
      > [frontend-6595944855-rsw4m server] panic: environment variable "RATING_SERVICE_ADDR" not set
      > [frontend-6595944855-rsw4m server] goroutine 1 [running]:
      > [frontend-6595944855-rsw4m server] main.mustMapEnv(0xc0003820a8, 0xb065ef, 0x13)
      > [frontend-6595944855-rsw4m server]      /go/src/github.com/GoogleCloudPlatform/microservices-demo/src/frontend/main.go:297 +0x10b
      > [frontend-6595944855-rsw4m server] main.main()
      > [frontend-6595944855-rsw4m server]      /go/src/github.com/GoogleCloudPlatform/microservices-demo/src/frontend/main.go:149 +0xae9
 - deployment/frontend failed. Error: container server terminated with exit code 2.
9/11 deployment(s) failed
make: *** [Makefile:26: cluster] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -12,40 +12,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.-PHONY: cluster deploy deploy-continuous logs check-env check-service
+.-PHONY: cluster deploy deploy-continuous logs destroy check-env check-service
 
 ZONE=us-west1-a
 CLUSTER=cloud-ops-sandbox-${USER}
 
 cluster: check-env
-	gcloud services enable container.googleapis.com
-	EXISTING_CLUSTER=`gcloud container clusters list --zone=${ZONE} --format="value(name)" --filter="name:${CLUSTER}"`
-	if [ "${EXISTING_CLUSTER}" != "" ]; then \
+	gcloud services enable container.googleapis.com; \
+	EXISTING_CLUSTER=`gcloud container clusters list --zone=${ZONE} --format="value(name)" --filter="name:${CLUSTER}"`; \
+	if [ "$${EXISTING_CLUSTER}" != "" ]; then \
     	gcloud container clusters delete ${CLUSTER} --project=${PROJECT_ID} --zone=${ZONE} --quiet; \
-	fi
-
+	fi; \
 	gcloud beta container clusters create ${CLUSTER} \
-		--project=${PROJECT_ID} --zone=${ZONE} \
-		--machine-type=n1-standard-2 --num-nodes=2 \
+		--project=$${PROJECT_ID} --zone=${ZONE} \
+		--machine-type=e2-standard-4 --num-nodes=4 \
 		--enable-stackdriver-kubernetes \
 		--scopes https://www.googleapis.com/auth/cloud-platform
-	cd ./terraform/istio && \
-	./install_istio.sh
-	skaffold run --default-repo=gcr.io/${PROJECT_ID}/sandbox -l skaffold.dev/run-id=${CLUSTER}-${PROJECT_ID}-${ZONE}
-	kubectl set env deployment.apps/frontend RATING_SERVICE_ADDR=http://some.host
+	./terraform/istio/install_istio.sh; \
+	skaffold run -f ./skaffold.yaml --default-repo=gcr.io/$${PROJECT_ID}/sandbox -l skaffold.dev/run-id=${CLUSTER}-$${PROJECT_ID}-${ZONE}
 
 deploy: check-env
-	echo ${CLUSTER}
-	gcloud container clusters get-credentials --project ${PROJECT_ID} ${CLUSTER} --zone ${ZONE}
-	skaffold run --default-repo=gcr.io/${PROJECT_ID}/sandbox -l skaffold.dev/run-id=${CLUSTER}-${PROJECT_ID}-${ZONE} --status-check=false
+	echo ${CLUSTER}; \
+	gcloud container clusters get-credentials --project $${PROJECT_ID} ${CLUSTER} --zone ${ZONE}; \
+	skaffold run --default-repo=gcr.io/${PROJECT_ID}/sandbox -l skaffold.dev/run-id=${CLUSTER}-$${PROJECT_ID}-${ZONE} --status-check=false; \
 	kubectl set env deployment.apps/frontend RATING_SERVICE_ADDR=http://some.host
 
 deploy-continuous: check-env
-	gcloud container clusters get-credentials --project ${PROJECT_ID} ${CLUSTER} --zone ${ZONE}
+	gcloud container clusters get-credentials --project $${PROJECT_ID} ${CLUSTER} --zone ${ZONE}; \
 	skaffold dev --default-repo=gcr.io/${PROJECT_ID}/sandbox --status-check=false
 
 logs: check-service
 	kubectl logs deployment/${SERVICE}
+
+destroy:
+	EXISTING_CLUSTER=`gcloud container clusters list --zone=${ZONE} --format="value(name)" --filter="name:${CLUSTER}"`; \
+	if [ "$${EXISTING_CLUSTER}" != "" ]; then \
+    	gcloud container clusters delete ${CLUSTER} --project=${PROJECT_ID} --zone=${ZONE} --quiet; \
+	fi
 
 check-env:
 ifndef PROJECT_ID
@@ -60,3 +63,4 @@ check-service:
 ifndef SERVICE
 	$(error SERVICE is undefined. Enter a service name (e.g. SERVICE=frontend))
 endif
+

--- a/kubernetes-manifests/frontend.yaml
+++ b/kubernetes-manifests/frontend.yaml
@@ -61,6 +61,8 @@ spec:
             value: "checkoutservice:5050"
           - name: AD_SERVICE_ADDR
             value: "adservice:9555"
+          - name: RATING_SERVICE_ADDR
+            value: "http://some.host"
           resources:
             requests:
               cpu: 100m

